app/javascript/packs/ajax_ops.js

```diff
@@ -91,7 +98,86 @@ function handle_ajax(event) {
           });
         }
       });
+    } else if (event.target === deleteMemberButton) {
```

Comment:
so in general, this mega else if block could probably be cleaned up with a `switch` statement and I would abstract all the logic into different functions based on the case to make it more readable.